steps/oasys/login.ts

```diff
@@ -0,0 +1,10 @@
+import { Page, expect } from '@playwright/test'
+
+export const login = async (page: Page) => {
+    await page.goto(process.env.OASYS_URL)
+    await expect(page.locator('#loginbodyheader > h2')).toHaveText('Login')
+    await page.fill('#P101_USERNAME', process.env.OASYS_USERNAME!)
+    await page.fill('#P101_PASSWORD', process.env.OASYS_PASSWORD!)
```

Comment:
Could you update the example `.env` file in the [README.md](https://github.com/ministryofjustice/hmpps-probation-integration-e2e-tests/blob/main/README.md?plain=1#L51) with sample values for these?
```
OASYS_URL
OASYS_USERNAME
OASYS_PASSWORD
```

Also we'll need to add the real values into the pipeline as secrets.  You can do this by creating Repository Secrets here: https://github.com/ministryofjustice/hmpps-probation-integration-services/settings/secrets/actions.

Once you've created the secrets we need to update the pipeline to reference them, here: https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/.github/workflows/end-to-end-tests.yml#L59-L72

Happy to help you through this process tomorrow if needed.